### PR TITLE
pretty print cart question file

### PR DESCRIPTION
**Pull request by christophmluscher**
dumps the cart question in a readable format

### CODE REVIEW

File: cart/questions.py [hunk 1]
```diff
@@ -287,5 +292,10 @@ def process_questions(root, questions):
         return root
 
     def write_to_file(self, file):
```
Comment (curufinwe) [inline cart/questions.py:294]: This override is not necessary.

File: cart/questions.py [hunk 2]
```diff
@@ -98,8 +98,13 @@ def get_questions(self):
...
-        tree = ET.ElementTree(self.get_questions())
-        tree.write(file, encoding="unicode")
+        from xml.dom.minidom import parseString
```
Comment (curufinwe) [inline cart/questions.py:101]: There is already a write_xml function with a prettify option in util.py. Please use that.

**Review by curufinwe — CHANGES_REQUESTED** [review]

File: cart/questions.py (hunk 1 above)
Comment (christophmluscher) [reply cart/questions.py:290]: do you mean to completely remove `write_to_file`? I am not sure where that is used. Probably only CartEstimateJob.

Comment (michelwi) [reply cart/questions.py:290]: As `CMUCartQuestions` inherits from `BasicCartQuestions` and the same implementation is already present in `BasicCartQuestions`, this function `write_to_file` will override the exact same function `write_to_file` from `BasicCartQuestions` with exactly the same implementation.

Therefore, deleting the function here will lead to the exact same behavior.

Comment (christophmluscher) [reply cart/questions.py:290]: But the function `write_to_xml` belongs to `PythonCartQuestions` and not to `CMUCartQuestions`. And `PythonCartQuestions` does not inherit from `BasicCartQuestions`

**Review by curufinwe — APPROVED** [review]

**Pull request merged**